### PR TITLE
Add IMMLA 

### PR DESCRIPTION
**Pull request by VladimirKuzlo**
#531

### CODE REVIEW

File: people/vitaly_stepanov.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Personal data
+![vitaly stepanov photo](photo/vitaly_stepanov.jpg)
```
Comment (Desser) [inline people/vitaly_stepanov.md]: please, correct path

Comment (VladimirKuzlo) [reply people/vitaly_stepanov.md]: Done

**Comment by VladimirKuzlo** [comment]
done

File: projects/immla.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+[Website](https://immla.io/)  
+Founded: 2017  
+Currency: IML  
+ICO: Start: 15. Sep 2017  
+End: 15. Oct 2017
+## Staff
+Executive Director: [Kirill Tulenev](../people/kirill_tulenev.md)   
+Co-founder & Information Director: [Vyacheslav Neunyvakin](../people/vyacheslav_neunyvakin.md)   
+Co-founder: [Mikhail Astakhov](../people/mikhail_astakhov.md)  
+Development Director: [Vitaly Stepanov](../people/vitaly_stepanov.md)
```
Comment (Desser) [inline projects/immla.md]: vitalii_stepanov.md

**Pull request merged**